File: src/core/jupiter/core/metrics/direction.ts
import { MetricDirection } from "@jupiter/webapi-client";

export function metricDirectionName(direction: MetricDirection): string {
  switch (direction) {
    case MetricDirection.NONE:
      return "None";
    case MetricDirection.UP_IS_GOOD:
      return "Up Is Good";
    case MetricDirection.DOWN_IS_GOOD:
      return "Down Is Good";
  }
}

File: src/core/jupiter/core/apps/metrics/direction.py
"""The direction for a metric."""

from jupiter.framework.value import EnumValue, enum_value


@enum_value
class MetricDirection(EnumValue):
    """The direction for a metric - whether up or down is good."""

    NONE = "none"
    UP_IS_GOOD = "up_is_good"
    DOWN_IS_GOOD = "down_is_good"
